com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/BuildCloudDeployModelStep.java

```diff
@@ -79,8 +79,10 @@ protected StepPhase executeStep(ProcessContext context) {
         // Build a list of applications for deployment and save them in the context:
         List<Module> modulesCalculatedForDeployment = calculateModulesForDeployment(context, deploymentDescriptor, mtaArchiveModules,
                                                                                     deployedModuleNames, mtaModules);
-
-        getStepLogger().debug(Messages.MODULES_TO_DEPLOY, secureSerializer.toJson(modulesCalculatedForDeployment));
+        List<String> moduleJsons = modulesCalculatedForDeployment.stream()
+                                                                 .map(module -> secureSerializer.toJson(module))

```

Comment:
Method reference could be used: secureSerializer::toJson

Reply:
10x!